3i-deploy/recipes/deploy-nodejs-CLI.rb

```diff
@@ -58,6 +58,12 @@
     action :delete
   end
 
+  execute 'build' do
+    command "cd #{deploy[:deploy_to]}/current && npm run build"
+    only_if { deploy['environment_variables']['REQUIRES_BUILD'] == 'true' }
```

Comment:
For my education:

Does the `only_if` mean we'll run the entire "execute `build`" section only if `REQUIRES_BUILD` is true, or does it mean we'll only do the `command` line at 63?  

Just wondering how/if line 64 is impacted by the `only_if`.